apps/video/js/view.js

```diff
@@ -504,6 +504,22 @@
     }
   }
 
+  function seekEnded() {
+    updateSlider();
+    if (pendingSeekPosition) {
```

Comment:
My bad! Thanks for the review and info about the linter. I'll install these modules and update the patch ASAP.